DisguiseUnityRenderStream/Runtime/PipelineAbstraction/PresenterInputUGUI.cs

```diff
@@ -0,0 +1,82 @@
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Disguise.RenderStream
+{
+    /// <summary>
+    /// Attaches to this <see cref="GameObject"/>'s <see cref="UnityEngine.EventSystems.BaseInputModule"/>
+    /// to convert the mouse coordinates to the specified <see cref="Presenter"/>.
+    /// </summary>
+    class PresenterInputUGUI : BaseInput
```

Comment:
This naming is a bit confusing because it's not related to the `Presenter` class.

Reply:
Changed to `UGUIInputForPresenter` and `UITKInputForPresenter`